sql/wsrep_mysqld.cc

```diff
@@ -2870,6 +2870,57 @@ static void wsrep_RSU_end(THD *thd) {
   thd->disable_binlog_guard.reset();
 }
 
+bool thd_enter_apply_monitor(THD* thd) {
```

Comment:
return value is not evaluated by the caller

Reply:
Changed it to `void` as thd->killed will be handled by the caller functions.